packages/desktop/views/dashboard/settings/views/security/index.js

```diff
@@ -1,6 +1,4 @@
 export { default as AppLock } from './AppLock.svelte'
```

Comment:
Not technically a part of this PR, but `AppLock` is not an accurate name... `ProfileLock` would be at least slightly more accurate

Reply:
Yeah I agree will add it to my lists of tasks to make.